Bartosz Furmanek

BartekFurmanek opened the pull request:
Zadanie na 4.5

BartekFurmanek committed:
zadania na 4.5

DocentSzachista commented on projekt_do_pracy_na_labach/fiszki_backend/Dockerfile:
```diff
@@ -1,2 +1,39 @@
 FROM python:3.10
 
+WORKDIR /usr/src/app
```
Polecam sobie WORKDIR uprościć, wystarczy do zwykłego `app/`. To jest tylko komentarz jb, nie wpływa na ocenę

DocentSzachista reviewed (commented):
Jak na razie 4.5. Jak chcesz zapraszam do zadania na 5.0

DocentSzachista committed:
Update lab-01.md

DocentSzachista committed:
Wykomentowanie zmiennej BACKEND_URL, dodanie portu który jest udostępniany

Teraz docker-compose będzie działał. 

TLDR:
- docker compose tworzy defaultową siec, po której jeżeli udostepnicie jakikolwiek port, będzie on dostępny pod adresem localhost
- Aby sposób ze zmienną zadziałał, wystarczy podmienić link na http://localhost:8000 
- W domyśle zadania było aby użyć nazwy serwisu i żeby aplikacja frontendowa nawiązała połączenie po DNS'ie. (Ale coś nie pykło z apką flutterową).

BartekFurmanek committed:
Zadanie na 5.0

BartekFurmanek commented on projekt_do_pracy_na_labach/fiszki_backend/Dockerfile:
```diff
@@ -35,5 +35,4 @@ CMD ["python", "main.py"]
 #docker rm mongo -f
 #docker run --network=siec -v mongo_volume:/data/db --name mongov_2 -d mongodb/mongodb-community-server
 #docker exec -it mongov_2 bash
-#Wynik: Dane testowe
```
Usunięte przez pomyłkę

BartekFurmanek commented:
Nie jestem do końca pewny czy frontend poprawnie działa.

DocentSzachista committed:
Dodanie instrukcji numer dwa do ocenienia

DocentSzachista committed:
Poprawki

DocentSzachista committed:
Merge pull request #30 from DocentSzachista/boss-man-consulting

Dodanie labów 2

DocentSzachista committed:
Drobne poprawki w słowie pisanym i obrazek

DocentSzachista committed:
Dodanie zadanek do lab 3, niedokończona wersja

DocentSzachista committed:
Dodanie opisów

DocentSzachista committed:
Aktualizacja rozpiski

BartekFurmanek committed:
Poprawki do zadania na 5.0

BartekFurmanek committed:
Ostatnia poprawka na 5.0

BartekFurmanek commented:
Teraz wszystko działa poprawnie. Musiałem zainstalować inny obraz mongo, i zrobić parę poprawek w kodzie.

DocentSzachista commented:
Jutro przetestuję ale wygląda moim zdaniem git, jak nie będę miał uwag to zamknę PR'a i wrzucę 5.0

DocentSzachista committed:
Merge pull request #31 from DocentSzachista/boss-man-consulting

Drobne poprawki w słowie pisanym i lab 3 (lab3 niedokończone jeszcze)

BartekFurmanek committed:
Merge pull request #1 from DocentSzachista/master

Lab2

BartekFurmanek committed:
Zadanie na 3.5 i 4.0

DocentSzachista commented:
Ocenkę za lab_1 wrzuciłem. (5.0) za lab_2 wezme sie w weekend.

BartekFurmanek committed:
Lab2 5.0

BartekFurmanek commented:
Na zajęciach pokazywałem błąd w minikubie: _Unable to resolve the current Docker CLI context "default": context "default": context not found: open C:\Users\HpX360\.docker\contexts\meta\37a8eec1ce19687d132fe29051dca629d164e2c4958ba141d5f4133a33f0688f\meta.json: The system cannot find the path specified._
Ogólnie minikube działał poprawnie mimo tego błędu, ale znalazłem rozwiązanie. Wystarczy po uruchomieniu minikuba użyć komendy **docker context use default**.

DocentSzachista commented:
5.0 i plusik wejdzie za rozwiazanie :p
